index.html

```diff
@@ -169,11 +169,13 @@ <h3 class="languages-h3">Skills</h3>
     <section class="touch">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     </section>
-    <article class="form">
-        <input type="text" placeholder="Full name">
-        <input type="email" placeholder="Email address">
-        <textarea name="" id="" cols="30" rows="10">Enter text here</textarea>
-    </article>
+
+    <form action="https://formspree.io/f/xpznodbr" method="POST">
+        <input type="text" placeholder="Full name" required maxlength="30">
+        <input type="email" placeholder="Email address" required>
+        <textarea name="message" id="" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
+    </form>
+
     <article class="touch-button">
         <button type="button">Get in touch</button>
         <ul class="icon">
```

Comment:
- [ ] Good Job for addressing suggested change requested by the previous reviewer but when I submit the form with `button` nothings happen this because of the `action` performed on `button` with method `post` to integrate with [formspree service](https://formspree.io/html) So Kindly warp `button` with in `form` tag and make sure that form submitted correctly.

Reply:
Fixed